Samples/20-Shadows/ShadowsApp.cs

```diff
@@ -244,7 +244,7 @@ protected override void Draw(GameTimer gt)
             CommandQueue.ExecuteCommandList(CommandList);
 
             // Present the buffer to the screen. Presenting will automatically swap the back and front buffers.
-            SwapChain.Present(0, PresentFlags.None);
+            SwapChain.Present(1, PresentFlags.None);
```

Comment:
What's the motivation behind this change?

Reply:
Just revert this ;)